pkg/utils/utils.go

```diff
@@ -16,3 +18,17 @@ func ParseSymbol(address uint64, table *helpers.KernelSymbolTable) *helpers.Kern
 	hookingFunction.Owner = strings.TrimSuffix(hookingFunction.Owner, "]")
 	return hookingFunction
 }
+
+// ErrorFuncName displays a given error prefixed by the current (caller) fn name
+func ErrorFuncName(e error) error {
```

Comment:
Nice addition, but I think it might be better centralized at the logger level. @geyslan I believe we have something similar there right?

Reply:
Yep, I suggest @geyslan picks this and move to the logger in another PR, meanwhile this PR can use it as its good for debugging (saved me 2 times).

Reply:
Reason (for @geyslan): AFAIK, Logger provides file and function and line but it does not concatenate errors from frame to frame as this does, which is good so I can just return "err" when it happens and it is still useful.

Reply:
I think this function is already in the right place (utils). Perhaps naming it `EnrichErr()` can be better.